build.sbt

```diff
@@ -164,7 +164,10 @@ lazy val docs = (project in file("apso-docs"))
     mdocOut := (ThisBuild / baseDirectory).value,
 
     mdocVariables := Map(
-      "VERSION" -> "0.25.1" // This version should be set to the currently released version.
+      "VERSION" ->
+        // This is reading the most recent header from the CHANGELOG that looks like a version, assuming that's the most
+        // recent version to refer to in the docs.
+        IO.readLines(file("CHANGELOG.md")).flatMap("""^## \[([0-9.]+)\]""".r.findFirstMatchIn).head.group(1)
```

Comment:
I am fine with this approach. But as food for thought, could we reason about the returned value in comparison to what's in `version.sbt`? It would be nice to be able to catch an error like a wrong header in the md file.

Reply:
Maybe, but that's hard to solve in general because `version.sbt` holds the "current" version and this `VERSION` should include the last published version, so we would need some mechanism able to compare versions (assuming a scheme and snapshot versions). Since an update to this `VERSION` variable results in an update to the README, I think mistakes are easy to spot as is.